README.md

```diff
@@ -21,7 +21,7 @@
 
 ### Important Notes
 
-1. **YOU MUST READ THE [Contributors Guide](https://github.com/go-gitea/gitea/CONTRIBUTING.md) BEFORE STARTING TO WORK ON A PULL REQUEST**.
+1. **YOU MUST READ THE [Contributors Guide](./CONTRIBUTING.md) BEFORE STARTING TO WORK ON A PULL REQUEST**.
```

Comment:
I don't have a strong opinion on that. If you say lgtm feel free to merge it like it is